lib/teiserver_web/controllers/admin/user_controller.ex

```diff
@@ -952,9 +952,23 @@ defmodule TeiserverWeb.Admin.UserController do
   @spec relationships(Plug.Conn.t(), map) :: Plug.Conn.t()
   def relationships(conn, %{"id" => id}) do
     user = Account.get_user!(id)
+    id = user.id
```

Comment:
Special thanks to @geekingfrog for fixing this bug.
Id was a string, not an integer.